indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -165,6 +180,9 @@ export default connect(
     hasOwnedRooms: userSelectors.hasOwnedRooms(state),
   }),
   dispatch => ({
+    // ATTENTION: this is **intentionally** passed as a prop, despite not being used,
```

Comment:
s/ATTENTION/XXX (most editors highlight FIXME, TODO and XXX in comments) ;)

Reply:
Mine doesn't 😉

Reply:
Isn't `XXX` meant for thing which is not quite ready?

Reply:
looks like there might be an extension for it: microsoft/vscode#9899

Reply:
I always interpreted it as "heads-up"

SO seems to agree: https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

Reply:
Other sources disagree, and to me it looks more like a FIXME or a TODO, so I'll leave "ATTENTION", which seems to be pretty self-explanatory. You can always configure your IDE to highlight those 😉